File: 1-Easy/The Descent/the-descent.cs
using System;

class Player
{
    static void Main(string[] args)
    {
        const int NUMBER_OF_MOUNTAINS = 8;
        
        // game loop
        while (true)
        {
            int maxHeight = 0;
            int maxIndex = 0;

            for (int i = 0; i < NUMBER_OF_MOUNTAINS; i++)
            {
                // maximum selection BUT the index matters
                int mountainHeight = int.Parse(Console.ReadLine());
                if(maxHeight < mountainHeight)
                {
                    maxHeight = mountainHeight;
                    maxIndex = i;
                }
            }
            Console.WriteLine(maxIndex);
        }
    }
}

File: 1-Easy/The Descent/the-descent.ts
const NUMBER_OF_MOUNTAINS: number = 8;

// game loop
while (true) {
    let maxHeight: number = 0;
    let maxIndex: number = 0;
    for (let i = 0; i < NUMBER_OF_MOUNTAINS; i++) {
        // maximum selection BUT the index matters
        const mountainHeight: number = parseInt(readline());
        if(maxHeight < mountainHeight) {
            maxHeight = mountainHeight;
            maxIndex = i;
        }
    }

    console.log(maxIndex);
}